modules/ztp-image-based-upgrade-generate-seed-image.adoc

```diff
@@ -0,0 +1,220 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-seed-generation_{context}"]
+= Generating a seed image with the {lcao}
+
+Use the {lcao} to generate the seed image with the `SeedGenerator` CR. The {lcao} will do the following:
+
+// seed cluster, some things are cluster specific that are not included
```

Comment:
What about any 3rd party parts like drivers that are installed with KMM?

Reply:
out of scope

Reply:
Half of production deployments use 3rd party drivers or libraries. This is not the answer we can tell them. You are giving them excuse to dump RH and go with a different vendor.